.github/workflows/ci.yml

```diff
@@ -87,20 +87,26 @@ jobs:
       working-directory: ${{github.workspace}}/dependencies
       run: sed -i 's/OpenCASCADE_LIBRARIES [A-Za-z0-9;]\+/OpenCASCADE_LIBRARIES TKSTEP;TKSTEPAttr;TKSTEP209;TKSTEPBase;TKIGES;TKBool;TKMesh;TKXSBase;TKBO;TKPrim;TKShHealing;TKTopAlgo;TKBRep;TKGeomAlgo;TKGeomBase;TKG3d;TKG2d;TKMath;TKernel/' occt_install/lib/cmake/opencascade/OpenCASCADEConfig.cmake
 
-    # TODO rename cache 25/11
     - name: Cache VTK
       id: cache-vtk
       uses: actions/cache@v2
       with:
         path: dependencies/vtk_install
-        key: vtk-unversioned-${{matrix.vtk_version}}-${{matrix.os}}
+        key: vtk-${{matrix.vtk_version}}-${{matrix.os}}
 
-    - name: Setup VTK
+    - name: Setup VTK and clone bare
       if: steps.cache-vtk.outputs.cache-hit != 'true'
       working-directory: ${{github.workspace}}/dependencies
       run: |
         mkdir vtk_build
         mkdir vtk_install
+        git clone --bare https://github.com/Kitware/VTK.git bare
```

Comment:
meh, keeping the long sha is better than doing extra work imho.

Reply:
this extra work is done only when building VTK, not when using the cache.

Reply:
Still not worth it ?

Reply:
It adds complexity to the CI. You think it worth it?

Reply:
I think this is fine to add if that helps us managing the CI

Reply:
ok then